.pipelines/cni/pipeline.yaml

```diff
@@ -1,5 +1,11 @@
 pr: none
-trigger: none
+trigger:
+  tags:
+    include:
+    - dropgz/*
+    - azure-cni/*
+    - azure-ipam/*

```

Comment:
should trigger for azure-cns release as well

Reply:
azure-cns and azure-cni are not submodules, so they're covered by the `v*` tag trigger